core/src/Ribasim.jl

```diff
@@ -28,6 +27,12 @@ TimerOutputs.complement!()
 include("validation.jl")
 include("solve.jl")
 include("config.jl")
+
+# namespaced config.jl to prevent clashes
+const Config = C.Config
```

Comment:
I don't think I fully understand why you are adding an alias instead of `using .C: Config, Solver, algorithm`

Reply:
That requires the package C (or config) to be added to the Project.toml (and Manifest) I believe, and I don't want to go down that rabbithole.

Reply:
Do you mind if I take over this PR and try?